lib/factory_girl/linter.rb

```diff
@@ -15,9 +15,10 @@ def lint!
       end
     end
 
-    private
-
     attr_reader :factories_to_lint, :invalid_factories
+    private     :factories_to_lint, :invalid_factories
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.